nav2_system_tests/src/planning/test_planner_node.cpp

```diff
@@ -37,27 +38,56 @@ class RclCppFixture
 
 RclCppFixture g_rclcppfixture;
 
-TEST_F(PlannerTester, testSimpleCostmaps)
-{
-  std::vector<TestCostmap> costmaps = {
-    TestCostmap::open_space,
-    TestCostmap::bounded,
-    TestCostmap::top_left_obstacle,
-    TestCostmap::bottom_left_obstacle,
-    TestCostmap::maze1,
-    TestCostmap::maze2
-  };
-
-  ComputePathToPoseResult result;
-
-  for (auto costmap : costmaps) {
-    loadSimpleCostmap(costmap);
-    EXPECT_EQ(true, defaultPlannerTest(result));
-  }
-}
+// #TODO(orduno) This test throws a fastrtps error most of the time when bringing down navfn
+// TEST_F(PlannerTester, testPlannerTransitions)
+// {
+//   EXPECT_NO_THROW(
+//     activate();
+//     nav2_util::bringup_lifecycle_nodes("/navfn_planner");
+//     nav2_util::bringdown_lifecycle_nodes("/navfn_planner");
+//     deactivate();
+//   );
+// }
+
+// #TODO(orduno) Once testing with random points stabilizes, re-enable this
```

Comment:
What's unstable about random point tests?

Reply:
Added more detail and created #1100.